docs/graphql/core/api-reference/schema-metadata-api/table-view.rst

```diff
@@ -242,13 +289,19 @@ Custom Root Fields
 
 .. _set_table_custom_fields:
 
-set_table_custom_fields
------------------------
+set_table_custom_fields (deprecated)
```

Comment:
Should we also say in [this table](https://deploy-preview-5719--hasura-docs.netlify.app/graphql/core/api-reference/schema-metadata-api/index.html#request-types) that this is depracated?

Reply:
I think yes, but would like to hear what @tirumaraiselvan also has to say

Reply:
Doesn't it say already? https://github.com/hasura/graphql-engine/pull/5719/files#diff-e8eabc9c3c339a6063c7b2c29b6fdb009017d9e0459be2978aee7617f31c9afeR115

Or do you mean add an extra column called deprecated to the table? I don't think that is necessary at this point. Maybe when we have few more deprecated APIs.

Reply:
I added the `(deprecated)` indicator yesterday. Earlier, there was nothing. So, Marion wanted some sort of indicator that the API is deprecated